tox.ini

```diff
@@ -50,6 +50,7 @@ setenv =
     PYTHONPATH = {env:PYTHONPATH:}:{toxinidir}/server/lib:{toxinidir}/lib
 deps =
     -r{toxinidir}/test-requirements.txt
+    -r{toxinidir}/server/test-requirements.txt
```

Comment:
Interesting. Should you be including `/agent/test-requirements.txt` as well? (Oddly, it only has `ansible`; does 0.69 really need ansible as a test dependency?)

Reply:
It strikes me that `agent/test-requirements.txt` really should include `pytest`...how "fortunate" that `server/test-requirements.txt` pulls that in for us....

Reply:
Why?

There are no "agent" `pytest` tests, are there?  Is the `pytest` command using `./server/lib/pbench/test/unit`?

Reply:
Indeed; so 0.69 has no agent pytests at all, even though it has this minimal `agent/test_requirements.txt` just to confuse the unwary reader? Fine. It wouldn't hurt, then, to just delete that file to avoid confusion, but it's not worth another pass.